serverless.yml

```diff
@@ -20,9 +20,10 @@ provider:
       - ${env:SUBNET_ID_B, 'subnetIdB'}
       - ${env:SUBNET_ID_C, 'subnetIdC'}
   environment:
-    RDF4J_SERVICE_URL: ${env:RDF4J_SERVICE_URL, cf:rdf4jLoadBalancerStack.RDF4JServiceUrl, 'http://localhost:8080'} 
+    RDF4J_SERVICE_URL: ${env:RDF4J_SERVICE_URL, cf:rdf4jLoadBalancerStack.RDF4JServiceUrl, 'http://localhost:8080'}
     RDF4J_USER_NAME: ${env:RDF4J_USER_NAME, 'rdf4j'}
     RDF4J_PASSWORD: ${env:RDF4J_PASSWORD, 'rdf4j'}
+    CMR_BASE_URL: ${env:CMR_BASE_URL, 'https://cmr.earthdata.nasa.gov'}
```

Comment:
You should also update deploy_bamboo.sh to include this new env var and update bamboo with the variable value.

Reply:
Done, added https://cmr.sit.earthdata.nasa.gov

Reply:
SIT, probably want to be prod since all the data is pointing to prod